lib/portage/process.py

```diff
@@ -61,13 +61,13 @@
     if os.path.isdir(_fd_dir):
         break
     else:
-        _fd_dir = None
+        _fd_dir = ""
```

Comment:
I think this change makes the code worse.

Reply:
Can you elaborate on why you think using `""` makes it worse?

Reply:
Setting it to `None` conveys that we are don't have a valid value to use. If we were to accidentally pass None to a function that expects a valid path, we would likely get a helpful error at runtime.

Setting it to `""` looks like it might just be a mistake, and the behavior of functions that accept a path equal to empty string is less predictable.

Reply:
Makes sense, I'll drop that change.

Reply:
Reverted.